.github/workflows/v2-beta.md

```diff
@@ -4,7 +4,10 @@
 
 ## Installation
 
-> You **must** uninstall `sfdx` before you install `sf` (v2), or you'll get an error.  This is true for all installation methods (npm, installers, and TAR files.)
+> **Warning**  
+> You **must** uninstall `sfdx` before you install `sf` (v2), or you'll get an error. See instructions for your installation methods below.
+> 
+> **Context:** We created an alias for `sfdx` within `sf` (v2) so that you would not have to update all of your scripts from `sfdx` to `sf`. The concequence is a one-time uninstall of `sfdx-cli` to prevent this collision.
```

Comment:
```suggestion
> **Context:** We created an alias for `sfdx` within `sf` (v2) so that you don't need to update your scripts from `sfdx` to `sf`. The consequence is a one-time uninstall of `sfdx-cli` to prevent this collision.
```